grafana-plugin/src/pages/integration/Integration.tsx

```diff
@@ -311,6 +314,65 @@ class Integration extends React.Component<IntegrationProps, IntegrationState> {
     }
   }
 
+  renderAlertmanagerV2MigrationHeaderMaybe(alertReceiveChannel: AlertReceiveChannel) {
+    if (!alertReceiveChannel.alertmanager_v2_migrated_at) {
+      return null;
+    }
+
+    const alertID = `alertmanager_v2_alert_hidden_${alertReceiveChannel.id}`;
+    if (getItem(alertID)) {
+      return null;
+    }
+    const onAlertRemove = () => {
+      setItem(alertID, true);
+      this.forceUpdate();
+    };
+
+    const migratedAt = moment(alertReceiveChannel.alertmanager_v2_migrated_at).toString();
+    const docsURL = `https://grafana.com/docs/oncall/latest/integrations/${alertReceiveChannel.integration.replace(
+      '_',
+      '-'
+    )}`;
+
+    return (
+      <div className="u-padding-top-md">
+        <Alert
+          severity="warning"
+          onRemove={onAlertRemove}
+          title={
+            (
+              <VerticalGroup>
+                <Text type="secondary">
+                  This legacy integration was automatically migrated at {migratedAt}. It now relies on Alertmanager's
+                  grouping and autoresolution mechanism.
+                </Text>
+                <Text type="secondary">Here are the steps you need to take to ensure a smooth transition:</Text>
+                <Text type="secondary">
+                  1. Check and adjust integration templates, as they were dropped back to default values during the
+                  migration.
+                </Text>
+                <Text type="secondary">
+                  2. Check and adjust integration routes so that they match the new payload shape.
+                </Text>
+                <Text type="secondary">
+                  3. Check and adjust outgoing webhooks that use alerts from this integration so that they match the new
+                  payload shape.
+                </Text>
+                <Text type="secondary">
+                  Refer to{' '}
+                  <a href={docsURL} target="_blank" rel="noreferrer">
+                    the docs
```

Comment:
shouldn't we show it as link so: `<Text type="link">the docs</Text>`?

Reply:
good catch! addressed in https://github.com/grafana/oncall/pull/3722/commits/a776c0d77b059e2960886adf51b37c48eea455a6